rkt/metadata_service.go

```diff
@@ -383,10 +384,18 @@ func mergeAppAnnotations(im *schema.ImageManifest, cm *schema.PodManifest) types
 func handleAppAnnotations(w http.ResponseWriter, r *http.Request, pm *schema.PodManifest, im *schema.ImageManifest) {
 	defer r.Body.Close()
 
+	n := mux.Vars(r)["app"]
+	an, err := types.NewACName(n)
```

Comment:
I believe you mean here:
https://github.com/coreos/rkt/pull/1052/files#diff-0a939c0d8e5b66c4a2e56cc2d890adb4R407